source/agora/consensus/Ledger.d

```diff
@@ -1485,6 +1468,61 @@ public class Ledger
     }
 }
 
+/// The Ledger class held by a node
```

Comment:
Due to the large size of  this file is it maybe better to create `NodeLedger.d` file?

Reply:
Check the linked PR, it does that in one of the last commits, but first I need to break down the dependencies.

Reply:
Great minds ...